.gitlab-ci.yml

```diff
@@ -25,6 +24,18 @@ unit_tests_julia1.9:
   tags:
     - cpuonly
 
+unit_tests_releases:
+  extends: .untit_test_template
+  parallel:
+    matrix:
+      - JULIA_VERSION: ["1.6", "1.7", "1.8", "1.9"]
+  image: julia:$JULIA_VERSION
+
+unit_tests_nightly:
+  extends: .untit_test_template
+  image: julia:1.10-rc
```

Comment:
wouldn't it be better to use the image `julia:nightly`, so there is less maintenance effort?

Reply:
There is no `julia:nightly`. Now, I use the same baseimage like the official julia container and install the nightly version at the begin of each test job.

Reply:
Excellent! I like this solution.